### DISCUSSION TITLE
Interpolation decay for user interaction vs. camera target setting

### DISCUSSION THREAD

**Discussion by DominikDa** [google/model-viewer#3498] [Ideas]
When setting camera target / camera orbit by an event, I feel that interpolation decay makes the camera transition feel more natural. In the user interaction, however, the interpolation decay feels like the model is reacting too slow.

Is there a way to integrate a second property to set the interpolation decay at different levels for user-induced camera movements and for event-induced camera movements?

**Comment by elalish** [comment]
No, and since they can happen concurrently, it's not really possible. However, you can change the decay dynamically; set it long in your event handler, then set it short again after a delay.

**Reply by DominikDa** [reply]
I should have thought about that. Great idea, thanks!
